packages/ui/src/components/Banner/index.tsx

```diff
@@ -59,13 +126,18 @@ export const Banner = ({
   image,
   className,
 }: BannerProps) => {
+  const defaultImage =
+    size === 'small' ? defaultIllustrationSmall : defaultIllustration
+
   const [opened, setOpened] = useState(true)
 
   if (!opened) return null
 
   return (
     <Container variant={variant} size={size} className={className}>
-      {image}
+      <ImageStack size={size} justifyContent="center">
+        {image ?? <img src={defaultImage} alt="" />}
```

Comment:
```suggestion
        {image ? <img src={defaultImage} alt="" /> : null}
```

Also should be 
```suggestion
        {image ? <img src={image ?? defaultImage} alt="" /> : null}
```

no  ?

Reply:
@matthprost So what's the point of passing `image` if we are always going to show the defaultImage ?

Reply:
Alright, that's me I'm dumb ... my bad